### PR TITLE
新建 文件夹 fork 测试

### PR DESCRIPTION
**Pull request by FromCt**
this  is not hello world !!!

### CODE REVIEW

File: from_ct/readMe [hunk 1]
```diff
@@ -0,0 +1,3 @@
+pull request 前需要
+git pull upstream master
```
Comment (JuoJuo) [inline from_ct/readMe]: please chang this line

File: from_ct/readMe [hunk 2]
```diff
@@ -0,0 +1,3 @@
+pull request 前需要
```
Comment (JuoJuo) [inline from_ct/readMe]: please use english

Comment (JuoJuo) [inline from_ct/readMe]: please fuck

Comment (FromCt) [reply from_ct/readMe]: fuck

**Comment by FromCt** [comment]
Boss , I have changed it.

**Pull request closed without merging**